ayodelevm/PostIt
# #150580005 Reset Password

#### What does this PR do?
- it adds reset password feature to post-it app

#### Description of Task to be completed?
- On request to reset password, this task first verifies that the users email exists, if it does, a unique link is generated and sent to the users email, which then redirects the user to a page where they can reset their password

#### Any background context you want to provide?
- Nil

#### How should this be manually tested?
- Clone the repository
- Run `npm install` to install dependencies
- Run `npm start` to start up the server
- Navigate to `http://localhost:3002/`
- Create an account and logout of the account
- Go to the login page
- Click on reset password link
- Enter your email
- Check your email for the link sent to you and open it
- enter a new password and login to your account with the new password

#### What are the relevant pivotal tracker stories?
#150580005 Reset Password

#### Screenshots (if appropriate)
- Nil

#### Questions:
- Nil
